src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <app-category-edit
+      :category="targetCategoryName"
+      :error-message="errorMessage"
+      :done-message="doneMessage"
+      @update-value="updateValue"
+      @click="updateCategoryName"
+    />
+  </div>
+</template>
+
+<script>
+import { CategoryEdit } from '@Components/molecules';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    errorMessage() {
+      return this.$store.state.categories.errorMessage;
+    },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
+    targetCategoryName() {
+      return this.$store.state.categories.targetCategory.name;
+    },
+  },
+  created() {
+    this.$store.dispatch('categories/getCategoryDetails', this.$route);
```

Comment:
[imo]

`this.$route` を引数として渡すと上記の記述をパッと見たときに何をしているのかがわかりづらいと個人的に思います。`this.$route`がどんなふうに使われているかがわかりにくいからです。

以下のような記述なら「カテゴリの詳細を id から取得しているんだな」というのがぱっと見でわかりやすいと思います。

```suggestion
    this.$store.dispatch('categories/getCategoryDetails', this.$route.params.id);
```

Reply:
修正しました！